.travis.yml

```diff
@@ -5,12 +5,6 @@ language: python
 python:
   - 2.7
 
-env:
-  - TOXENV=django18
-  - TOXENV=django19
-  - TOXENV=django110
-  - TOXENV=django111
-
```

Comment:
Please revert this change -- the django version should be irrelevant.

Reply:
Done. Agreed about Django; I just thought that running the jobs in parallel might be causing the erratic errors, but after trying it I don't think Travis runs them with the same Elastic instance.